tests/test_benchmarks/test_imagenet_c.py

```diff
@@ -0,0 +1,57 @@
+import numpy as np
+import pytest
+from pytest import approx
+
+from brainio_base.assemblies import BehavioralAssembly
+from brainscore.benchmarks.imagenet import Imagenet2012
+from brainscore.benchmarks.imagenet_c import Imagenet_C_Individual, Imagenet_C_Noise, Imagenet_C_Blur, Imagenet_C_Weather, Imagenet_C_Digital
+from brainscore.model_interface import BrainModel
+
+#@pytest.mark.slow
```

Comment:
this is commented out -- following slack something breaks if you comment it in, but it's still not clear to me what exactly goes wrong